core/src/test/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisherTests.java

```diff
@@ -137,6 +138,49 @@ public void unknownFailureExceptionIsIgnored() {
 		verifyZeroInteractions(appPublisher);
 	}
 
+	@Test(expected = IllegalArgumentException.class)
+	public void emptyMapCausesException() {
+		Map<Class<? extends AuthenticationException>,
+				Class<? extends AbstractAuthenticationFailureEvent>> mappings = new HashMap<>();
+		publisher = new DefaultAuthenticationEventPublisher();
+		publisher.setAdditionalExceptionMappings(mappings);
+		ApplicationEventPublisher appPublisher = mock(ApplicationEventPublisher.class);
```

Comment:
Are the last two lines of this test necessary? Let's go ahead and remove any unnecessary lines of code.